package.json

```diff
@@ -85,34 +85,15 @@
 	"browserslist": [
 		"extends @wordpress/browserslist-config"
 	],
-	"resolutions": {
-		"**/@automattic/calypso-build/node-sass": "^4.13.1",
-		"**/gulp/gulp-cli/yargs/yargs-parser": "^20.0.0",
-		"**/ini": "^1.3.7",
-		"**/lodash": "4.17.21",
-		"**/node-fetch": "^2.6.1",
-		"**/node-notifier": "^8.0.1",
-		"**/tar-stream/bl": "^4.0.0",
-		"**/terser-webpack-plugin/serialize-javascript": "^5.0.0"
-	},
 	"dependencies": {
+		"chalk": "4.1.1",
```

Comment:
Only a couple of deps we hadn't declared. That's not too bad, methinks.

Reply:
I didn't try running all the tools manually, but we don't have too many that are written in js.